.changeset/breezy-crabs-remain.md

```diff
@@ -0,0 +1,6 @@
+---
+"@microsoft/atlas-site": patch
```

Comment:
This is an addition in functionality, so a `minor` change is appropriate.

Reply:
I see. And thank you for making commit to fix this.